<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WkfMilStdIconPrefWidget</class>
 <widget class="QWidget" name="WkfMilStdIconPrefWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>458</width>
    <height>455</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Mil-Std 2525D</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="label_2">
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>此页面上的设置会在所有 AFSIM 应用程序之间共享</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="mUseSymbologyBox">
     <property name="text">
      <string>使用 Mil-Std 2525D 符号</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="mVelocityVectorBox">
     <property name="text">
      <string>显示速度矢量</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label">
     <property name="text">
      <string>以下所列的团队将根据指定的身份进行分类。  </string>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout" stretch="20,20,1">
     <item>
      <widget class="QComboBox" name="mCategoryBox"/>
     </item>
     <item>
      <widget class="QComboBox" name="mTeamBox">
       <property name="editable">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="mAddButton">
       <property name="text">
        <string>新增</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="wkf::TopLevelDropTreeWidget" name="mTeamMapping">
     <property name="acceptDrops">
      <bool>false</bool>
     </property>
     <property name="dragEnabled">
      <bool>false</bool>
     </property>
     <attribute name="headerVisible">
      <bool>false</bool>
     </attribute>
     <column>
      <property name="text">
       <string notr="true">1</string>
      </property>
     </column>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="mRemoveButton">
       <property name="text">
        <string>移除</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>wkf::TopLevelDropTreeWidget</class>
   <extends>QTreeWidget</extends>
   <header>WkfTopLevelDropTreeWidget.hpp</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
